hints/hints.go

```diff
@@ -0,0 +1,76 @@
+package hints
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/manifoldco/torus-cli/ui"
+)
+
+var commandHints map[string][]string
+
+// Init generates the commandHints map
+func Init() {
+	commandHints = map[string][]string{
+		"allow": {
+			"Grant additional access to a team or role using `torus allow`",
+		},
+		"context": {
+			"Your linked organization and project are found in .torus.json after running `torus link`",
+		},
+		"deny": {
+			"Restrict access for a team or role using `torus deny`",
+		},
+		"invites approve": {
+			"Approve multiple invites quickly with `torus worklog resolve`",
+		},
+		"invites send": {
+			"Invite another user to collaborate in your organization with `torus invites send`",
+		},
+		"link": {
+			"Define an organization and project for your current working directory using `torus link`",
```

Comment:
👍  I think this is better language, if we're not using this in the quickstart we should